llvm/docs/LangRef.rst

```diff
@@ -701,6 +701,9 @@ information. Attaching section information to an external declaration is an
 assertion that its definition is located in the specified section. If the
 definition is located in a different section, the behavior is undefined.
 
+LLVM allows an explicit code model to be specified for globals. If the
+target supports it, it will emit globals in the code model specified.
```

Comment:
`it will emit globals in the code model specified, overriding the code model used to compile the translation unit. The allowed values are "tiny", ..., "large". This may be extended in the future to specify global data layout that doesn't cleanly fit into a specific code model.`